department-of-veterans-affairs/abd-vro
# Do isort in Github workflow instead of Gradle

## What was the problem?

There was some discrepancy in the rules being performed by the Gradle isort task and the Precommit isort task, making them irreconcilable either at commit time or at testing. There were also many files that were not currently compliant with precommit hooks and it was cluttering other PR

## How does this fix it?[^1]
Moves the Gradle isort task for `service-python` project into a Github workflow step to achieve consistency in the rules applied and removes the Gradle isort task.

Runs the command `pre-commit run --all-files` and commits a portion of the results (some are not formatted correctly afterwards according to flake8 and will be addressed in a later PR)

## How to test this PR
- Step 1 Testing in the Github workflow 

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
